### PR TITLE
feat(wbgt): Added WGBT and tests.

### PR DESCRIPTION
**Pull request by janeliutw**
issue #34 
This PR adds WGBT and tests for WGBT. One thing I wanted to bring up is that the code for outdoor WGBT seems to be different from the reference paper. I wasn't sure if I am supposed to rewrite the code and update the formula according to the formula in the reference paper, but for now, I just cleaned up the legacy code and document without changing the logic in the code (see more details in the comments)

### CODE REVIEW

File: ladybug_comfort/wbgt.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    K. Blazejczyk et al. in 2014 [1] with the natural wet bulb temparature
+    (Tnwb) based on the code written by Nick Burns [2].
+
+    Note:
+        [1] Heat stress and occupational health and safety spatial
+        and temporal differentiation, K. Blazejczyk, J.Baranowski, A. Blazejczyk,
+        Miscellanea geographica regional studies on development, Vol. 18,
+        No. 1, 2014
+        [2] https://github.com/nickb-/Calculating-WBGT/blob/master/relaxation_Tw/
+        working_code/wbgt.R
```
Comment (janeliutw) [inline ladybug_comfort/wbgt.py:25]: I copied this hyperlink from the document in the legacy code, however, this repo is no longer accessible. This repo is where the legacy code used to calculate natural wet bulb temperature (Tnwb), which seems to be different from the formula in the paper in [1]. (see comment for line 38)

File: ladybug_comfort/wbgt.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        mrt: Radiant temperature [C]
+        ws: Wind speed [m/s]
+        vp: Vapour pressure [hPa]
+
+    Returns:
+        outdoor WBGT
+
+    """
+    # Natural wet bulb temperature
+    tnwb = 1.885 + 0.3704 * ta + 0.4492 * vp
```
Comment (janeliutw) [inline ladybug_comfort/wbgt.py:38]: the formula I found in the [paper](https://content.sciendo.com/configurable/contentpage/journals$002fmgrsd$002f18$002f1$002farticle-p61.xml) [1] is 
`Tnw = -9.27522 + 0.70196 * Ta + 0.30338 * vp + 0.07823 * RH`
Maybe either formulas works but I just don't know, so I thought I would bring this discrepancy up and see if  anyone has ideas?

File: ladybug_comfort/wbgt.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+"""Utility functions for calculating the Wet Bulb Globe Temperature (WBGT)."""
+from __future__ import division
+
+import math
+
+
+def wbgt_outdoors(ta, mrt, ws, vp):
+    """
+    The wet-bulb globe temperature (WBGT) is a type of apparent temperature,
+    and is widely used heat stress index globally (ISO 7243). It is used to
```
Comment (chriswmackey) [inline ladybug_comfort/wbgt.py:11]: We usually prefer to have the docstrings formatted with a single concise line at the top. Then, a line break after which you can include whatever other information you want.

File: ladybug_comfort/wbgt.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+
+def wbgt_warning_category(wbgt):
+    """
+    Get the category of warning associated with a given Wet Bulb Globe
+    Temperature(WBGT).
+
+    Categories are based on the US National Weather Service(NWS) issue the
+    following suggested actions and impact prevention:
+    * 0 = No Warning.
```
Comment (chriswmackey) [inline ladybug_comfort/wbgt.py:97]: I think we may need an extra line break before the bullet points in order to get it to display correctly in the online docs. [See here for an example with heat index](https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/hi.py#L65).

File: ladybug_comfort/wbgt.py (hunk 2 above)
Comment (chriswmackey) [reply ladybug_comfort/wbgt.py:38]: FYI, ladybug core is a dependency of ladybug-comfort and so you can just access the `saturated_vapor_pressure` method by putting the following at the top of the file:
```
from ladybug.psychrometrics import saturated_vapor_pressure 
```

File: ladybug_comfort/wbgt.py [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+    return wbgt_od
+
+
+def wbgt_indoors(ta, tdp, ws):
+    """
+    Used for indoors, or when solar radiation is negligible.
+    WBGT indoor formula by Bernard et al. (1999), formula from: "Calculating
+    Workplace WBGT from Meteorological Data: A Tool for Climate Change Assessment",
+    Lemke, Kjellstrom, 2012
```
Comment (chriswmackey) [inline ladybug_comfort/wbgt.py:52]: Could we format the reference with a `Note:` section?

**Review by chriswmackey — COMMENTED** [review]
Thank you very much for adding this, @janeliutw .  This is a big help.  Most of my comments are small styling issues about the docstrings.  My only large comment is that it would be great to have the `wbgt_outdoors` function take relative humidity as an input instead of vapor pressure and I pointed you towards a ladybug-core method that you can use to get vapor pressure from ta and rh.  Let me know what you think.

**Comment by chriswmackey** [comment]
I incorporated these edites as part of [this PR](https://github.com/ladybug-tools/ladybug-comfort/pull/190). So I am going to close this one out.

**Comment by janeliutw** [comment]
Sorry the notifications were buried in the pile of email, didn't see these message until now. Thank you for finishing it up!

**Pull request closed without merging**